src/parser/getCSSRules.ts

```diff
@@ -6,7 +6,7 @@
  * @return {Object} CSS rules of this document
  */
 export function getCSSRules(doc) {
-  let styles = doc.getElementsByTagName('style'),
+  const styles = doc.getElementsByTagName('style'),
```

Comment:
this is surely an error
you can move the definition of `i` to the loop and remove `len` if is used only once